Makefile

```diff
@@ -19,3 +19,6 @@ html-proofer:
 
 shell:
 	@docker-compose run --rm php bash
+
+update-sub-modules:
+	git submodule sync && git submodule update --init --remote --force
```

Comment:
Is this the equivalent of doing `git pull` on the submodule, without changing its branch?

Reply:
It is. Note it's just included as a convenience method.

`git submodule sync --recursive` was the first command I went for. This one being here might help people to update the submodules more regularly; or at least documents getting the latest from a branch / ref (if ref is used it will just checkout the submodule(s) as refs never change).